src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -4895,6 +4895,19 @@ components:
           items:
             type: string
             format: text
+        validation_errors:
+          description: object that contains validation errors bound to their respected fields - the object's key
+            represents the field where the validation error has happened and the key's value is the validation error
+          type: object
+          additionalProperties:
+            type: array
+            description: array containing the validation errors of a field
+            items:
+              type: string
+              format: text
+          example:
+            clientAdd.client.memberCode: ["NoPercent"]
```

Comment:
I think that we should document the different validation error codes for the end users. Both our own custom errors (NoPercent, NoBackslashes, etc) and standard bean validation errors (NotNull, Size, etc - this should probably be just a link to documentation elsewhere, possible some section of Bean Validation API specification?). 

Not sure what is a good way to do this, in openapi specification, or maybe somewhere in https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#19-management-rest-apis ?

Similar documentation is missing for error codes and metadata, but we already have a ticket about that https://jira.niis.org/browse/XRDDEV-665

Reply:
I think it could be a good idea to add them into the official documents (e.g. the one you linked or https://github.com/nordic-institute/X-Road/blob/develop/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#192-executing-rest-calls). Then we can just add a link to the document in the `validation_errors` description field.

Reply:
Added a [chapter](https://github.com/nordic-institute/X-Road/blob/XRDDEV-1001-validation-error-structure/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#194-validation-errors) to the X-Road User Guide and a mention about it into the API definition.